component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -2083,6 +2096,7 @@ protected void triggerEvent(String userName, String tenantDomain, String userSto
     }
 
     /**
+     * @deprecated This is method should be replaced by {@link #triggerEvent(AuthenticatedUser, Map)}.
```

Comment:
shall we add the deprecated version as well?

Reply:
fixed